examples/cpu_features/WORKSPACE

```diff
@@ -10,3 +10,31 @@ local_repository(
 load("@rules_android_ndk//:rules.bzl", "android_ndk_repository")
 
 android_ndk_repository(name = "androidndk")
+
+# Import rules_android
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+RULES_ANDROID_COMMIT = "a54904b056250a213117cee9287c996cd042f388"
+
+RULES_ANDROID_SHA = "bf08c364e6d0946d09b485aee388557bba3bdb5f15323f73c2723d07d3503ba6"
+
+http_archive(
+    name = "build_bazel_rules_android",
```

Comment:
I believe that rules_android uses the NDK in a few places -- so this might create a circular dependency (at least conceptually -- rules_android is using the native android_ndk_repository for the moment). If this is just for the platform definitions, it might actually make more sense to move those to rules_android_ndk, because rules_android itself doesn't really need to know about those platforms.